src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] tasks = new Task[100];
+    public static int taskCount = 0;
+
+    public static void printGreeting() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+    }
+    public static void printConfused() {
+        System.out.println("Could you say that again?");
+    }
+    public static void printExit() {
```

Comment:
I think adding a space after every method would make your code seem more readable and neater.